ansible/prod.yml

```diff
@@ -21,3 +21,4 @@
     - docker
     - rke
     - rancher
+    - newrelic
```

Comment:
Do we need some sort of switch to be able to disable it (e.g. for our internal testing)?

Reply:
We don't need it here since Newrelic services does not get deployed when newrelic licensekey is not passed.